assets/dev-env.lando.template.yml.ejs

```diff
@@ -160,6 +160,10 @@ services:
     services:
       image: elasticsearch:8.18.2
       command: /usr/local/bin/docker-entrypoint.sh
+      deploy:
+        resources:
+          limits:
+            memory: 1GB
```

Comment:
This seems like a sane default value but probably it may not be enough for larger apps.